scripts/bgfx.idl

```diff
@@ -1054,16 +1054,16 @@ func.VertexLayout["end"]
 	"void"
 
 --- Returns relative attribute offset from the vertex.
-func.VertexLayout.getOffset { const , cpponly }
```

Comment:
Make this `cppinline` and remove `cpponly` on methods that are `cppinline`.

Reply:
> remove `cpponly` on methods that are `cppinline`.

This will break all the other bindings, is that OK?

Reply:
I'm saying rename `cppheadronly` into `cppinline` and then when use `cpponly` or `cppinline`, not both.

Fix other bindings to respect the `cppinline` you introduced.

Reply:
Basically after you introduce `cppinline` and fix other bindings generators to respect `cppinline`, after you regenerated IDL you should not see any changes in other languages at all, if you see changes something is not correct.

Reply:
> I'm saying rename `cppheadronly` into `cppinline` and then use `cpponly` or `cppinline`, not both.
> 
> Fix other bindings to respect the `cppinline` you introduced.

OK but it might take a while since my arm is currently broken.